Headers/DebugServer2/Target/Windows/Process.h

```diff
@@ -59,6 +59,7 @@ class Process : public ds2::Target::ProcessBase,
   }
 
 public:
+  ErrorCode getFileLoadAddress(std::string const &file_path, Address &address);
```

Comment:
You might want to make this have a similar interface to `enumerateSharedLibraries`, just for consistency.